cmd/blockparser/blockparser.go

```diff
@@ -114,9 +132,17 @@ func Parser(sig chan bool) error {
 
 	fmt.Println("Building the .txos file...")
 	fmt.Println("Starting from block:", tipnum)
+
+	//Receive signal from stopParse so the for loop can break
+	var stop bool
+	go func() {
```

Comment:
I guess this works!  But it seems weird because stop is being affected in its own goroutine.

Instead, how about this at the bottom of the loop (line 159）
```
		select {
		case stop = <-stopGoing:
		default:
		}
```

this might actually do the exact same thing (they both seem to work) but this feels a lot safer / easier to understand for me as there are fewer concurrent routines happening.